docs/coding_conventions.md

```diff
@@ -0,0 +1,101 @@
+Coding conventions
+=================
+
+Formatting
+----------
+
+- **Length** of a line should not be longer than 80 characters. In the rare
+  cases when we need a longer line (e.g. in a preprocessor macro) use backslash
+  character to mark a continuation.
+- **Parameter list** is a frequent place, when a line tends to be longer than
+  the limit. Break the line and align parameters, as seen in the example below.
+- **Tab characters** are not welcome in the source. Replace them with spaces.
```

Comment:
It would be better if the _not_, or the _spaces_ is highlighted. Making **Tab characters** bold makes it looks like they should be used.